PrebidMobile/PrebidMobileRendering/Prebid/PBMCacheRenderers/InterstitialController.swift

```diff
@@ -64,40 +65,12 @@ public class InterstitialController: NSObject, PBMAdViewManagerDelegate {
     }
     
     @objc public func loadAd() {
```

Comment:
Please put a comment for this function

// TODO: provide a more relevant name for this function. 

It's out of the scope of this PR, but I want to return back later and solve the inconvenience of naming the function. Because, in fact, this function doesn't load anything, it initializes the internal class respectively to the received bid and other properties.